src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -247,27 +263,11 @@ internal class ConnectionManager : IConnectionManager {
                 }
             }
 
-            if (!string.IsNullOrEmpty(connectionInfo?.Path) && await TrySwitchBrokerAsync(connectionInfo)) {
-                return;
-            }
-
-            var connection = RecentConnections.FirstOrDefault(c => !c.IsRemote);
```

Comment:
Correct. They may or may not want local started and may or may not want us to try last remote since it may be down or they may be offline. In fact, they may have REPL up since last session but not intend to use R.